joomla/joomla-cms
# [4.0] Update joomla/database

### Summary of Changes

Updates joomla/database package.

### Testing Instructions

https://github.com/joomla/joomla-cms/pull/26867 is required for MySQLi to work.

1) Install an extra language. Select the language in user profile. Uninstall the language. Check that the language is removed from `params` column in `#__users` table.

2) Go to Menus -> Manage. Check that publish/unpublish/trashed menu item count is correct.

3) Check that menu rebuild function works without errors.

4) Browse around some other pages.

### Expected result

Works like before.

### Documentation Changes Required

No.
